changedetectionio/templates/watch-overview.html

```diff
@@ -27,18 +27,18 @@
 					{% endif %}
 				{% endfor %}
 			</div>
- -->			<div id="controls-top">
+			<div id="controls-top">
 				<ul id="post-list-buttons-top">
 					<span id="checkbox-functions" style="display: none;">
 						<li>
 							<a href="javascript:processChecked('recheck_selected', '{{ active_tag }}');" class="pure-button button-tag " title="Recheck Selected{%if active_tag%} in &quot;{{active_tag}}&quot;{%endif%}">Recheck</a>
-						</li>
+						</li><br>
 						<li>
 							<a href="javascript:processChecked('mark_selected_notviewed', '{{ active_tag }}');" class="pure-button button-tag " title="Mark Selected as Unviewed{%if active_tag%} in &quot;{{active_tag}}&quot;{%endif%}">Unviewed</a>
-						</li>
+						</li><br>
```

Comment:
Br's dont belong in <UL>

Reply:
Removed.  I added them for IE because I couldn't get display:-ms-grid to work, but I have it working now.

Reply:
Noting that the fixed position grid display of the checkbox functions needs IE11 if IE is used.